TF_Tooling_Web/src/Cursors/SvgCursor.ts

```diff
@@ -43,27 +43,29 @@ export class SVGCursor extends TouchlessCursor {
         svgRingElement.setAttribute('r', '15');
         svgRingElement.setAttribute('fill-opacity', '0');
         svgRingElement.setAttribute('stroke-width', '5');
-        svgRingElement.setAttribute('stroke', _darkCursor ? 'black' : 'white');
+        svgRingElement.setAttribute('stroke', darkCursor ? 'black' : 'white');
         svgRingElement.setAttribute(this.xPositionAttribute, '100');
         svgRingElement.setAttribute(this.yPositionAttribute, '100');
         svgRingElement.style.filter = 'drop-shadow(0 0 10px rgba(255, 255, 255, 0.7))';
         svgElement.appendChild(svgRingElement);
+        svgRingElement.id = 'svg-cursor-ring';
         this.cursorRing = svgRingElement;
 
         const svgDotElement = document.createElementNS('http://www.w3.org/2000/svg', 'circle');
         svgDotElement.classList.add('touchfree-cursor');
         svgDotElement.setAttribute('r', '15');
-        svgDotElement.setAttribute('fill', _darkCursor ? 'black' : 'white');
+        svgDotElement.setAttribute('fill', darkCursor ? 'black' : 'white');
         svgDotElement.setAttribute(this.xPositionAttribute, '100');
         svgDotElement.setAttribute(this.yPositionAttribute, '100');
         svgDotElement.setAttribute('opacity', '1');
         svgDotElement.style.transformBox = 'fill-box';
         svgDotElement.style.transformOrigin = 'center';
         svgDotElement.style.transform = 'scale(1)';
         svgDotElement.style.filter = 'drop-shadow(0 0 10px rgba(255, 255, 255, 0.7))';
+        svgDotElement.id = 'svg-cursor-dot';
         svgElement.appendChild(svgDotElement);
 
-        if (!_darkCursor) {
+        if (!darkCursor) {
```

Comment:
👍 I like the change to match convention of only using _ for unused variables

Reply:
I'm going to try and make these changes where ever I come across them in the codebase to slowly update it to fit "normal" Typescript rules